### DISCUSSION TITLE
Remove/convert a node without select the previous one

### DISCUSSION THREAD

**Discussion by AxelBriche** [ueberdosis/tiptap#2666] [General Questions]
Hi, I have 3 node extensions:

**Spoiler**
content: 'spoilerTitle spoilerContent'

**SpoilerTitle**
content: 'text*'
defining: true

**SpoilerContent**
content: 'block+'

When I remove all first characters inside the title, the node is not converted to a paragraph like on the [Details extension](https://tiptap.dev/api/nodes/details) and create a selection of the previous node.

**Current behavior:**

![CleanShot 2022-03-30 at 20 44 25](https://user-images.githubusercontent.com/1295251/160908673-4a371a6e-2d41-41b7-941b-a7a314dae3ba.gif)

Expected behavior:

![CleanShot 2022-03-30 at 20 48 00](https://user-images.githubusercontent.com/1295251/160908894-f2dc3f4a-26d1-43f1-9662-0536de926044.gif)

I have play with different content type, defining, atom, selectable, etc. and can't reproduce the expected behavior (remove my node and convert it's content to simple paragraph).

**Comment by philippkuehn** [comment]
At some point you have to register custom handlers for these cases. The details extension registers custom handlers for `Backspace`, `Enter`, `ArrowRight` and `ArrowDown` for example.
